redisinsight/ui/src/pages/home/components/form/TlsDetails.tsx

```diff
@@ -48,6 +88,24 @@ const TlsDetails = (props: Props) => {
     optionsCertsCA.push({
       value: cert.id,
       inputDisplay: cert.name,
+      dropdownDisplay: (
+        <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+          <div>{cert.name}</div>
+          <PopoverDelete
+            header={cert.name}
+            text="will be deleted from RedisInsight."
+            item={cert.id}
+            suffix={suffix}
+            deleting={certRef.current}
+            closePopover={closePopover}
+            updateLoading={false}
+            showPopover={showPopover}
+            handleDeleteItem={() => handleDeleteCaCert(cert.id)}
+            handleButtonClick={() => handleClickDeleteCert(cert.id)}
+            testid={`delete-ca-cert-${cert.id}`}
```

Comment:
could you please also add several ui tests, that we call telemetry and redux actions after click on delete button?

Reply:
I added the UI tests.